textarea/textarea.go

```diff
@@ -227,7 +227,7 @@ type Model struct {
 }
 
 // New creates a new model with default settings.
-func New() Model {
+func New() *Model {
```

Comment:
Agreed. This would break that code, and could potentially be left as is while still accomplishing the goal of the PR. I’ll fix this as soon as I can.